apps/design/backend/src/app.test.ts

```diff
@@ -261,28 +265,19 @@ test('update election info', async () => {
   expect(election.seal).toEqual('\r\n<svg>updated seal</svg>\r\n');
   expect(election.date).toEqual(new DateWithoutTime('2022-01-01'));
 
-  // empty string values do overwrite existing values
-  await apiClient.updateElectionInfo({
-    electionId,
-    type: 'primary',
-    title: '',
-    jurisdiction: '  ',
-    state: '',
-    seal: '',
-    date: new DateWithoutTime('2022-01-01'),
-  });
-
-  expect(await apiClient.getElection({ electionId })).toEqual(
-    expect.objectContaining({
-      election: expect.objectContaining({
+  // empty string values are rejected
+  await suppressingConsoleOutput(() =>
+    expect(
+      apiClient.updateElectionInfo({
+        electionId,
+        type: 'primary',
         title: '',
-        county: expect.objectContaining({
-          name: '',
-        }),
+        jurisdiction: '  ',
         state: '',
         seal: '',
```

Comment:
I think we're still accepting an empty seal based on my read of the schema

Reply:
Yeah, we do accept empty string as a seal value. Arguably we shouldn't, but I don't think we're going to change it right now.

Reply:
Ah forgot about that, cool!